Makefile

```diff
@@ -1,5 +1,5 @@
 CC= g++
-CFLAGS= -Wall -g -O2 -shared -fPIC -I/usr/include/mongo `pkg-config --cflags lua5.2` `pkg-config --cflags libmongo-client`
+CFLAGS= -Wall -fno-strict-aliasing -g -O2 -shared -fPIC -I/usr/include/mongo `pkg-config --cflags luajit` `pkg-config --cflags libmongo-client`
```

Comment:
Restore the old line, adding LUAVERSION variable. I'm interested in the details about the aliasing problem, if we can fix the code to remove it. Just annotate here the error given by the compiler, and the compiler version.
```
CFLAGS= -Wall -g -O2 -shared -fPIC -I/usr/include/mongo `pkg-config --cflags lua${LUAVERSION}` `pkg-config --cflags libmongo-client`
```